components/StudentPage/index.js

```diff
@@ -2,8 +2,11 @@ import React, { useState, useContext, useEffect } from "react";
 import SPTasks from "./SP-Tasks";
 import SPETStag from "./SP-ETStag";
 import SPDependents from "./SP-Dependents";
-import styles from "../../styles/StudentPage.module.css";
-import Chat from "../Chat";
```

Comment:
this line should be in this file

Reply:
My dude, its not my component

Reply:
its in there in main  but not in your branch for some reason